.github/workflows/trivy.yaml

```diff
@@ -31,23 +31,23 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Run Trivy vulnerability scanner in repo mode
-        uses: aquasecurity/trivy-action@master
-        with:
-          scan-type: 'fs'
-          ignore-unfixed: true
-          format: 'sarif'
-          output: 'trivy-results.sarif'
-          severity: 'LOW,MEDIUM,HIGH,CRITICAL'
+      # - name: Run Trivy vulnerability scanner in repo mode
```

Comment:
you need to build blobfuse2 before this step: `Run Trivy vulnerability scanner in repo mode`

Reply:
Removed this and added binary scan towards the end of this file